packages/gatsby/cache-dir/default-html.js

```diff
@@ -12,6 +12,7 @@ export default class HTML extends React.Component {
             name="viewport"
             content="width=device-width, initial-scale=1, shrink-to-fit=no"
           />
+          {this.props.gatsbyVersion && <meta name="generator" content={`Gatsby ${this.props.gatsbyVersion}`} />}
           {this.props.headComponents}
```

Comment:
Instead of adding it to the default html component (which is often overrided), set it as a component in headComponents

Reply:
I was so close to doing it this way instead 🙃 Will do it that way, one sec